CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# v6.0.0 (2025-02-07)
+
+## OS Changes
+* Add the ability to run custom transaction checks when committing transactions to Bottlerocket’s datastore ([#294])
+* Add support for `weak` settings values, which are deleted on update ([#294])
+* Only return `strong` settings-generators from the apiserver’s /metadata/settings-generator route ([#294])
+* Always delete and re-populate metadata on first boot or update ([#294])
+* Add support for dynamic settings-generators via the depth attribute ([#294])
+* Add NVIDIA Multi-Instance GPU (MIG) settings to nvidia-k8s-device-plugin ([#258])
+* Conditionalize source and mode in Bootstrap container template ([#335])
+* Update host-ctr dependencies ([#337])
+* Accept comment lines in boot config ([#361])
+
+## Build Changes
+
```

Comment:
Should we remove this heading?

Reply:
I wanted to but I noticed people were leaving it in.

Reply:
ohh my bad. I read it wrong. Let me get rid of it.